cfme/tests/networks/test_sdn_addresses.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from cfme import test_requirements
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.networks.views import FloatingIpView
+
+pytestmark = [
+    test_requirements.sdn,
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.provider([OpenStackProvider], scope="module")
+]
+
+
+@pytest.fixture()
+def network_manager(appliance, provider):
+    network_manager, = appliance.collections.network_providers.filter({"provider": provider}).all()
```

Comment:
Can you wrap this potential ValueError in a pytest.skip call, so it doesn't start throwing errors when there's more/less than 1 element in the list.

Reply:
@mshriver Is this something like you wanted? https://github.com/ManageIQ/integration_tests/pull/10033/files#diff-d0ee99cb7213e3f233490166c76aab34R15